gpu-functions-mojo/README.md

```diff
@@ -106,83 +106,65 @@ implement that in MAX.
 
     The function itself is very simple, running once per thread, adding each
     element in the two input vectors that correspond to that thread ID, and
-    storing the result in the output vector at the matching location. If a block
-    of threads is dispatched that is wider than the vector length, no work is
-    performed by threads past the end of the vector.
+    storing the result in the output vector at the matching location.
 
     ```mojo
     fn vector_addition(
-        lhs: LayoutTensor[float_dtype],
-        rhs: LayoutTensor[float_dtype],
-        out: LayoutTensor[float_dtype],
+        lhs_tensor: LayoutTensor[float_dtype, layout, MutableAnyOrigin],
```

Comment:
I've found `LayoutTensor[mut=True, float_dtype, layout]` to be more succinct and readable.

Reply:
Updated with that cleaner format throughout the examples.